### ISSUE TITLE
When i open two Projects at the same time then git in not working in second Project.

### ISSUE THREAD

**Issue by garvitjoshi** [Microsoft/vscode#26432]

**Comment by egamma** [comment]
@garvitjoshi please provide more detailed steps. What do you mean by 'open two projects at the same time'. What do you mean by 'git not working'.

**Comment by garvitjoshi** [comment]
![screenshot from 2017-05-11 12-33-29_li 5]
(https://cloud.githubusercontent.com/assets/15026629/25936959/765974dc-3647-11e7-975b-db4b18222f10.jpg)
Hello **@egamma** here I attach an image in which my two different projects are open in vscode. but one of them not showing my git branch(that is my second Project that means before that I already open different project and that is my first project ).

**Comment by joaomoreno** [comment]
Can you try out our [insider release](https://code.visualstudio.com/insiders)?

**Comment by garvitjoshi** [comment]
I want to say that sometimes it's working sometimes not. I think it's a small bug. and sure I'll check it in insider release. and one more thing when I install insider release I want to install all extension which was installed on my previous vscode and that is ridiculous for me. so, I think you can do one more thing that is login or signup in vscode and when a user install extension so he/she can access all his extension by the help of simple login in vscode means he/she can see it's installed extensions in vscode.

**Comment by joaomoreno** [comment]
Copy `~/.vscode/extensions` to `~/.vscode-insider/extensions`.

**Comment by garvitjoshi** [comment]
Thanks, buddy you save my lots of time. _for ~/.vscode/extensions to ~/.vscode-insider/extensions._

**Comment by Kwisatx** [comment]
I've got the same issue since I updated from 1.11.0 to 1.12.1.
It happens fairly systematically for me that the first instance of vs code functions correctly, but any further instances have the git issue.

So when running `code path_to_project1` and then `code path_to_project2` project1 has git functionnality and project2 doesn't.
A project started first:
![vs_code_git_functionning](https://cloud.githubusercontent.com/assets/1901676/25993797/9241e30e-370b-11e7-94c7-e3a38568826c.png)
The same project started second:
![vs_code_git_not_functionning](https://cloud.githubusercontent.com/assets/1901676/25993798/92463bca-370b-11e7-9d8b-f6c0c1d55dd3.png)

Interestingly enough, sometimes if I close the first instance, the second one will refresh and git will function again.

I tried the insider release though, and was no longer able to reproduce the issue.
